src/script_number.js

```diff
@@ -35,11 +35,11 @@ function decode (buffer, maxLength, minimal) {
 
 function scriptNumSize (i) {
   return i > 0x7fffffff ? 5
-  : i > 0x7fffff ? 4
-  : i > 0x7fff ? 3
-  : i > 0x7f ? 2
-  : i > 0x00 ? 1
-  : 0
+    : i > 0x7fffff ? 4
+      : i > 0x7fff ? 3
+        : i > 0x7f ? 2
+          : i > 0x00 ? 1
+            : 0
```

Comment:
@feross was this intentional in standard 11?

Reply:
This was not intentional.